msteams-platform/bots/how-to/conversations/send-proactive-messages.md

```diff
@@ -7,6 +7,153 @@ ms.author: anclear
 ---
 # Send proactive messages
 
+In some situations, a bot may need to send to the user a message that is not related to the current conversation. These types of messages are called **proactive messages**.
+
+Proactive messages can be useful in a variety of scenarios. These are some examples:  
+
+- The user previously asked the bot to monitor the price of a product, the bot can alert the user if the price of the product has changed.
+- A bot requires some time to compile a response to the user's question, it may inform the user of the delay and allow the conversation to continue in the meantime.
+
+## Best practices
+
+### Basic guidelines
```

Comment:
Again, this needs be framed with respect to **Teams**. Bots should probably not create a bunch of new conversations within a team channel all at once, but maybe a bot need to create a bunch of separate 1-1 conversations with various users in response to some stimulus (a poll maybe?). Check with Andrew what this list should look like.